autopush/settings.py

```diff
@@ -204,18 +206,35 @@ def message(self, value):
         """Setter to set the current message table"""
         self.message_tables[self.current_msg_month] = value
 
+    def add_tomorrow(self, today, this_month_table):
+        """check to see if tomorrow is a new month, in case we're a node that
+        has a slow clock."""
```

Comment:
Possible idea for a quick check to see if next months table was already added, since we know they sort ok...
```python
if sorted(self.message_tables.keys())[-1] != self.current_msg_month:
    return
```